### ISSUE TITLE
disable require-description-complete-sentence with inheritDoc

### ISSUE THREAD

**Issue by Prozilla** [gajus/eslint-plugin-jsdoc#1641]
## Motivation

I would like to use this rule, but I'm also using some [TypeDoc](https://typedoc.org/)-specific syntax which is being flagged by this rule. The options of this rule seem quite limited at the moment, so I think they could use an expansion.

## Current behavior

I have TypeDoc comments in my codebase which look like this:

```ts
/** {@inheritDoc SomeClass} */
```

This is a tag that tells TypeDoc to copy the documentation from `SomeClass`. It's being flagged by this plugin because it does not end with a period. I don't see any way I can keep my TypeDoc comments like this and also enforce this rule.

## Desired behavior

It would be nice if we could have more control over where the rule is applied. For example, there could be an option that makes the rule ignore certain comment lines if they start or end with a certain character.

## Alternatives considered

As mentioned before, I don't really see any proper alternatives. I can't remove these comments because that messes up my documentation and I also don't want to disable this rule (it's why I installed this plugin in the first place).

I noticed on TypeDoc's documentation that this tag can also be used as an inline tag instead, even though this might be a viable workaround, I still think it would be good to add some more control over this rule.
